docs/HowToBuild.html

```diff
@@ -22,13 +27,27 @@ <h3 class="categoryDetailTitle">Jump to...</h3>
     <li><a href="#DebuggingPlugins">Debugging Plugins</a></li>
     <li><a href="#HostingPlugins">Hosting Plugins</a></li>
 </ul>
+
 <h3 class="categoryDetailTitle">Useful Links</h3>
 <ul class="categoryDetailList">
     <li><a href="FormItJSAPI/group__mod__jsapi__formit.html" target="_blank">FormIt Core JS API</a></li>
     <li><a href="FormItJSAPI/group__mod__jsapi__wsm.html" target="_blank">FormIt Modeling Kernel (WSM) JS API</a></li>
     <li><a href="../index.html" target="_blank">Using Plugins + The Plugin Manager</a></li>
 </ul>
 
+<div class="categoryBlock" id="PluginPlayground">
+    <h2 class="categoryTitle">Your first plugin from the Plugin Playground</h2>
+    <div class="categoryDetailBlock">
+        <p>
+            Building a plugin can be intimidating at the beginning. To help with this, we've introduced a Plugin Playground, where you can write, test, debug, and see your plugin in action - never needing to leave the FormIt application.
```

Comment:
If you don't mind, can we change this to:

If you're building your first FormIt plugin, the Plugin Playground is a great place to start. Write, test, debug, and view your plugin directly inside FormIt for Web or Windows, then publish instantly to GitHub for others to try.